starter-core/src/main/java/io/micronaut/starter/feature/build/maven/templates/pom.rocker.raw

```diff
@@ -819,6 +822,14 @@ List<Property> properties
           <!-- Uncomment to enable incremental compilation -->
           <!-- <useIncrementalCompilation>false</useIncrementalCompilation> -->
           <annotationProcessorPaths combine.children="append">
+@if (features.contains("lombok")) {
+            <path>
+              <!-- must precede micronaut-inject-java -->
```

Comment:
This one is tricky. By default, now we inherit the annotation processor definitions from the parent POM, and the `combine.children="append"` attribute in line 824 will make the effective POM to contain [the parent elements](https://github.com/micronaut-projects/micronaut-core/blob/2.2.x/parent/build.gradle#L121-L130) first, then Lombok.

If Lombok needs to go first, then your feature needs to generate something like:

```xml
<annotationProcessorPaths combine.self="override">
  <!-- lombok -->
  <!-- micronaut-inject-java -->
  <!-- micronaut-validation -->
</annotationProcessorPaths>
```

Reply:
@alvarosanchez Do I need to have two `annotationProcessorPaths` each with the different attributes, or am I changing the attribute on the existing one with `combine.self="override"` for all the `<path> child elements?

Reply:
I think I answered my own question and updated the PR with a revised changeset.